### DISCUSSION TITLE
Error import React Native project

### DISCUSSION THREAD

**Discussion by phongdt0811** [openpgpjs/openpgpjs#1409] [Q&A]
Error return "Navigator.userAgent.indexOf"

React Native is not Web API, when compile RN not detect navigator.userAgent
I did use the trick at navigator.userAgent = "React Native" at beginning but it doesn't work.

Please help!

**Comment by larabr** [comment]
Hi @phongdt0811 , React Native does not implement the Crypto API, which means this library cannot be used with it, at least not out-of-the-box. You can find an old discussion here: https://github.com/openpgpjs/openpgpjs/issues/1007, but note that the referenced libraries seem to be unmaintained, and have open security issues.
